scan-framework/src/main/java/com/getbouncer/scan/framework/util/Layout.kt

```diff
@@ -66,6 +66,12 @@ fun minAspectRatioSurroundingSize(area: Size, aspectRatio: Float): Size {
     }
 }
 
+@CheckResult
```

Comment:
Can you please add a comment explaining what this does? It's not immediately obvious from the code.